config/global.ini.php

```diff
@@ -641,6 +641,9 @@
 ; By enabling this flag the header HTTP_X_FORWARDED_URI will be considered for the current script name.
 proxy_uri_header = 0
 
+; If set to 1 we use the last IP in the list of proxy IPs when determining the client IP.
+proxy_ip_read_last_in_list = 1
```

Comment:
Shouldn't we make `0` as default to keep bc. Or at least set this option automatically to `0` on update, to ensure it won't break any instances. Could maybe only do that if `proxy_client_headers` is set.

Reply:
It was specified to set it to 1 in the issue (I assume it adds security). cc @tsteur

Reply:
By default we want to keep existing behaviour and follow the RFC so we'd set it to `0` as otherwise it would be a breaking change. 

We probably wouldn't leave the default at 1 and change it for existing installs as there are often random issues with it and it would require an additional step when setting up proxy headers. I reckon we could create an issue though to do this in Matomo 5 and put it directly into the Matomo 5 milestone.

Reply:
fyi could tweak the description a bit.Like

> ; If set to 1, the last IP in the list of proxy IPs will be used when determining the client IP. Using the last IP can increase the security when you are using proxy headers in combination with a load balancer. By default, the first IP is read according to RFC7239 which may be required in some cases when the client sends the IP through a proxy header as well as the load balancer.

Reply:
FYI put https://github.com/matomo-org/matomo/issues/17202 in Matomo 5 milestone